lib/double_entry/reporting.rb

```diff
@@ -78,8 +78,8 @@ class AggregateFunctionNotSupported < RuntimeError; end
     # @raise [Reporting::AggregateFunctionNotSupported] The provided function
     #   is not supported.
     #
-    def aggregate(function, account, code, range, options = {})
-      Aggregate.formatted_amount(function, account, code, range, options)
+    def aggregate(function, account, code, range, filter: nil, partner_account: nil)
```

Comment:
Not having any idea how widespread the use of this gem is, how problematic would it be to introduce a breaking change into the public-facing API?